src/modlunky2/config.py

```diff
@@ -164,6 +167,16 @@ def from_path(cls, config_path: Path, exe_dir=None):
         # Packing
         obj.show_packing = config_data.get("show-packing", SHOW_PACKING_DEFAULT)
 
+        # Level Editor
+        obj.level_editor_tab = config_data.get("level_editor_tab")
+
+        obj.custom_level_editor_custom_save_formats = config_data.get(
+            "custom_level_editor_custom_save_formats"
+        )
+        obj.custom_level_editor_default_save_format = config_data.get(
+            "custom_level_editor_default_save_format"
+        )
```

Comment:
keys in the config are defined with `kebab-case`, not `snake_case`

Reply:
Done.